index.html

```diff
@@ -65,6 +65,73 @@ <h2 class="i-m">
             </li>
         </ul>
     </main>
+    <section class="works">
+        <h1 class="head-works">My Recent Works</h1>
+        <hr>
+        <section class="content">
+            <div class="project">
+                <img src="ressources/Ilustration/Prj.png" alt="">
+            </div>
+            <div>
+                <h2 class="head-description">Multy-Post Stories</h2>
+                <p class="project-description">A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required. has been the industry's 
+                    standard dummy text ever since the 1500s, when an unknown 
+                    printer took a standard dummy text.</p>
+                <ul class="programming-language">
+                    <li class="item">CSS</li>
+                    <li class="item">HTML</li>
+                    <li class="item">JavaScript</li>
+                    <li class="item">Webpack</li>
+                </ul>
+            </div>
+            <button class="button-project">
+                See Project
+            </button>
+        </section>
+    </section>
+    <section class="about-me">
+        <section>
+            <h1 class="about-title">About Me</h1>
+            <p class="about-description">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t
+                    hesitate to contact me.</p>
+            <button class="about-button">Get My Resume</button>      
```

Comment:
- [ ] please kindly apply the appropraite padding to the `get my resume` button so that it shows and appears correctly just like the figma design
- figma design
 ![nm](https://github.com/AndresEspi/portafolio/assets/89055502/bda6571f-5e1a-4002-9e49-4446a54ab035)
- your design
![fv](https://github.com/AndresEspi/portafolio/assets/89055502/da65859c-2474-4379-8891-41c32a915b84)